--- app/xml_cdr/xml_cdr_inc.php	2019-12-15 08:47:11.518709725 +0000
+++ app/xml_cdr/xml_cdr_inc.php.dlucio	2019-12-19 14:26:42.344223963 +0000
@@ -446,6 +446,9 @@ $archive_request = 'true';
 	if (permission_exists("xml_cdr_mos")) {
 		$sql .= "c.rtp_audio_in_mos, \n";
 	}
+	if (file_exists($_SERVER["PROJECT_ROOT"]."/app/billing/app_config.php")){
+		$sql .= "carrier_name, call_buy, call_sell_local_currency as call_sell, ";
+	}
 	$sql .= "(c.answer_epoch - c.start_epoch) as tta ";
 	if ($_REQUEST['show'] == "all" && permission_exists('xml_cdr_all')) {
 		$sql .= ", c.domain_name \n";
